src/frameworks/drupal7/faq.md

```diff
@@ -19,7 +19,7 @@ deployment hooks to automatically run the updates.
 
 ## How can I override the default robots.txt?
```

Comment:
There is no longer a default robots.txt, so this question and probably the first sentence of the answer need to change.

Reply:
Revised to clarify that it's really only about where to put `robots.txt` in the `drupal` build mode.